mono/metadata/appdomain.c

```diff
@@ -2210,8 +2211,9 @@ deregister_reflection_info_roots (MonoDomain *domain)
 #endif
 
 static guint32 WINAPI
-unload_thread_main (void *arg)
+unload_thread_main_inner (void *arg)
```

Comment:
If this is becoming the inner you could likely remove the WINAPI attribute.

Reply:
Oops, an oversight. Will remove.

Reply:
Removed in 2c8d866